### PR TITLE
[12.0][MIG] stock_inventory_verification_request

### PR DESCRIPTION
**Pull request by osimallen**
Migration of stock_inventory_verification_request to version 12

Depends on stock_inventory_discrepncy: https://github.com/OCA/stock-logistics-warehouse/pull/622

### CODE REVIEW

**Comment by bodedra** [comment]
:green_apple: Ready for review.

@lreficent

**Comment by bodedra** [comment]
@max3903

File: oca_dependencies.txt [hunk 1]
```diff
@@ -3,3 +3,4 @@
 product-attribute
 server-ux
 web
+stock_inventory_discrepancy https://github.com/ursais/stock-logistics-warehouse bodedra_12.0-mig-stock_inventory_discrepancy
```
Comment (max3903) [inline oca_dependencies.txt]: @osimallen You should remove this and wait for the module to be merged and rebase this branch.

**Review by max3903 — APPROVED** [review]

File: oca_dependencies.txt (hunk 1 above)
Comment (bodedra) [reply oca_dependencies.txt]: @max3903 I added those line. Yes will do.

**Comment by bodedra** [comment]
@lreficent Would you please review these.

**Review by LoisRForgeFlow — CHANGES_REQUESTED** [review]
LGTM, just remove the line in oca_dependencies as pointed by @max3903 

Thanks for the work :+1:

**Comment by bodedra** [comment]
Thanks @max3903 @lreficent Comment attended. Test case are failed in OCA/OCB stock_request_purchase module.

**Review by LoisRForgeFlow — APPROVED** [review]
Travis fixed, let's merge!

**Pull request merged**